src/index.ts

```diff
@@ -23,20 +23,29 @@ interface TelescopeNode {
 }
 
 interface Config {
+  /**
```

Comment:
remove duplicate comment and then remake not sure how this happened haha

Reply:
ah good catch - must have committed the autosuggestion twice!